### PR TITLE
small fix in dispatcher-library section

### PR DESCRIPTION
**Pull request by FriendlyLifeguard**
Small syntax fix in one of the code snippet examples.

- - -
This change is [Reviewable](https://reviewable.io/reviews/cairo-book/cairo-book/544)

### CODE REVIEW

**Review by enitrat — COMMENTED** [review]
Thank you! 
While we're at it, can you:
- rename `trait IContractB<TContractState>` -> `trait IContractA<TContractState>`

change
```
    #[generate_trait]
    #[abi(per_item)]
```
to a simple `#[abi(embed_v0)]

and remove the `#[external(v0)]` above the functions?

otherwise, it wont compile properly

File: listings/ch99-starknet-smart-contracts/listing_99_09/src/lib.cairo [hunk 1]
```diff
@@ -19,7 +19,7 @@ mod ContractA {
 
     #[generate_trait]
     #[abi(per_item)]
-    impl ContractA of IContractA {
+    impl ContractA of IContractB {
```
Comment (enitrat) [inline listings/ch99-starknet-smart-contracts/listing_99_09/src/lib.cairo:22]:
```suggestion
    impl ContractA of super::IContractA {
```

**Comment by FriendlyLifeguard** [comment]
Hello, I just committed the changes. I also made changes to the explanation to the code snippet as well!

**Review by enitrat — DISMISSED** [review]
Amazing 💯

**Pull request merged**